paddle/fluid/inference/anakin/convert/activation.cc

```diff
@@ -61,4 +61,5 @@ void ActivationOpConverter<TargetT, PrecisionT>::operator()(
 REGISTER_ANAKIN_OP_CONVERTER(sigmoid, SigmoidOpConverter);
 REGISTER_ANAKIN_OP_CONVERTER(tanh, TanhOpConverter);
 REGISTER_ANAKIN_OP_CONVERTER(swish, SwishOpConverter);
+REGISTER_ANAKIN_OP_CONVERTER(mish, MishOpConverter);
```

Comment:
You don't need to add mish in Anakin, since it doesn't support it.

Reply:
So shall I remove Mish from Anakin completely in all files that come under anakin?

Reply:
Done.